asyncio_pipe/__init__.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
+# coding: utf8
+
+""" Asynchronous multiprocessing communication. """
+
+import asyncio
+
+from multiprocessing import Pipe
+from multiprocessing.connection import Connection
+from typing import Any, Optional, Tuple
+
+
+class AsyncConnection(object):
```

Comment:
Additionally, I don't think using `asyncio.Connection` and `multiprocessing.Connection` in the same context is a huge concern, since most people are creating instances of `multiprocessing.Connection` using `multiprocessing.Pipe`. The main reason I see for importing `multiprocessing.Connection` directly would be to use the [static methods](https://docs.python.org/3.4/library/multiprocessing.html?highlight=process#module-multiprocessing.connection). But in that case, you could just import `multiprocessing` and `asyncio`, or write `from asyncio import Connection as AsyncConnection`. 

Again, Python already has `Queue`, `multiprocessing.Queue`, and `asyncio.Queue`.

Reply:
Alright, fair enough to me, my opinion was mostly oriented with `typing` where you import everything (but this case is easily covered by aliasing indeed). I will patch this ASAP